webui/react/src/pages/SignIn.tsx

```diff
@@ -110,7 +110,13 @@ const SignIn: React.FC = () => {
    * This will prevent the form from showing for a split second when
    * accessing a page from the browser when the user is already verified.
    */
-  if (queries.has('jwt') || info.externalLoginUri || !isAuthChecked) return null;
+  if (
```

Comment:
can we update the top comment?

Reply:
Yep good call, I'll add it as a 4th case